CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 <!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->
 
 ## Enhancements
-* Added BETA support for including `enforced` attribute to variable_set on create and update by @Netra2104 [#778](https://github.com/hashicorp/go-tfe/pull/778)
+* Added BETA support for including `priority` attribute to variable_set on create and update by @Netra2104 [#778](https://github.com/hashicorp/go-tfe/pull/778)
```

Comment:
Looks like go-tfe was already released with the name `enforced`: https://github.com/hashicorp/go-tfe/releases/tag/v1.35.0.
Is this right?

If this is the case we may need to make the update to `priority` backward compatibility so it doesn't break after upgrade.

Reply:
Had discussions and turns out it can be renamed without backward compatibility since the field is marked BETA.